pkg/cmd/openshift/openshift.go

```diff
@@ -109,14 +109,16 @@ func NewCommandOpenShift(name string) *cobra.Command {
 		Run:   cmdutil.DefaultSubCommandRun(out),
 	}
 
+	f := clientcmd.New(pflag.NewFlagSet("", pflag.ContinueOnError))
```

Comment:
I'm confused why we can't do the `f := clientcmd.New(cmds.PersistentFlags())` thing inside `NewCmdVersion`